core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -334,7 +381,9 @@ export class BaseUser {
    * This function navigates to the given URL.
    */
   async goto(url: string): Promise<void> {
-    await this.page.goto(url, {waitUntil: 'networkidle0'});
+    await this.page.goto(url, {
+      waitUntil: ['networkidle0', 'networkidle2', 'domcontentloaded', 'load'],
+    });
```

Comment:
While keeping both ‘networkidle0’ and ‘networkidle2’ is fine, ‘networkidle2’ might be less restrictive and just be redundant as anyway it will wait for the networkidle0 ?
 Similarly, ‘load’ and ‘domcontentloaded’ overlap, so can we choose the stricter condition which is 'load' ? 

Also, shall we go with ‘networkidle2’ to be on safer side because as I have seen some instance of perpetual network activity even post complete navigation? (not sure)